api/replication.storage/v1alpha1/volumereplication_types.go

```diff
@@ -36,6 +36,24 @@ const (
 	ConditionValidated = "Validated"
 )
 
+// These are valid messages for various
+// conditions and states of Volume
+// replication.
```

Comment:
```suggestion
// These are valid messages for various conditions and states of volume replication.
```

lets make it one line comment